### DISCUSSION TITLE
How to make Thai text readable in Variables window

### DISCUSSION THREAD

**Discussion by preechaw** [apache/netbeans#4474] [Q&A]
Hi.
I wonder if it's possible to make Variables window display Thai text correctly instead of rectangles.
I created a watch with "Thai text ข้อความภาษาไทย". All Thai text became rectangles.
![2022-08-07_22h41_26](https://user-images.githubusercontent.com/23615475/183299185-e521e06e-e96e-448e-887f-6604c77a9e54.png)

Thank you.

**Comment by mbien** [comment]
not sure. This looks like linux - so netbeans should be using UTF-8 which means we can exclude a bunch of possible issues. Is the String in your program also UTF-8? Its interesting that the title is displayed correctly.

Contributions are welcome in that area.
